cirrus/server/tests/test_telemetry.py

```diff
@@ -161,3 +162,10 @@ def test_ping(data):
 
     assert ping_spy.call_count == 1
     assert app.state.metrics.cirrus_events.enrollment_status.test_get_value() is None
+
+
+def test_instance_name_metric(mocker):
+    app.state.pings, _ = initialize_glean()
+    ping_spy = mocker.spy(app.state.pings.startup, "submit")
+    send_instance_name_metric()
+    assert ping_spy.call_count == 1
```

Comment:
unfortunately I don't find a way to do that,  I just checked that, I can test metrics, but since I am submitting it, it gets wiped out, I think in the integration test we should be able to do more detailed testing